### ISSUE TITLE
Tank Drive via two joysticks

### ISSUE THREAD

**Issue by AmazingSpanoMan** [kerfuffle/FTC-9533#3]
Can we make the left joystick control the left motor and the right control the right? 
May have a learning curve but will be much easier to control.

**Comment by kerfuffle** [comment]
Although I am not sure how I am going to start going about that it seems feasible.

**Comment by AmazingSpanoMan** [comment]
it would just be forward and backward on each thumbstick. Forward makes the
motor go forward back makes it go in reverse. not need for turning that
would be done by pushing up on one and down on another.

**Comment by madikasu** [comment]
K9TankDrive seems to be set up for this already. Can we use that code instead of the teleop code for motor control?

**Comment by kerfuffle** [comment]
Yeah we are able to use the code, gonna have to modify it tho to meet our robot specs.

**Comment by AmazingSpanoMan** [comment]
Can we get Madikasu on as a collaborator? Also, Kerfuffle, are you coming to the meeting today? I'd like to get that code going.

**Comment by kerfuffle** [comment]
Yeah, and Ill be there today, might be a little late. In regards to the motors, yeah that's possible.
